core/server/data/migrations/versions/2.2/3-insert-admin-api-client-role.js

```diff
@@ -0,0 +1,83 @@
+const logging = require('../../../../lib/common/logging');
+const merge = require('lodash/merge');
+const models = require('../../../../models');
+const utils = require('../../../schema/fixtures/utils');
+
+const _private = {};
+
+_private.addApiKeyRole = (options) => {
+    const message = 'Adding "Admin API Client" role to roles table';
+    const apiKeyRole = utils.findModelFixtureEntry('Role', {name: 'Admin API Client'});
+
+    return models.Role.findOne({name: apiKeyRole.name}, options)
+        .then((role) => {
+            if (!role) {
+                logging.info(message);
+                return utils.addFixturesForModel({
+                    name: 'Role',
+                    entries: [apiKeyRole]
+                }, options);
+            }
+
+            logging.warn(message);
+        });
+};
+
+_private.addApiKeyPermissions = (options) => {
+    const message = 'Adding permissions for the "Admin API Client" role';
+    const relations = utils.findRelationFixture('Role', 'Permission');
+
+    return utils.addFixturesForRelation({
+        from: relations.from,
+        to: relations.to,
+        entries: {
+            'Admin API Client': relations.entries['Admin API Client']
```

Comment:
Shouldn't the administrator role get's it's new permissions? 🤔 

>                "Administrator": {                  
                    "integration": "all",
                    "api_key": "all"

Reply:
Yep, I'll get on that. Do migrations have tests at all?

Reply:
no

Reply:
Pushed this in a separate migration